test/support/builders/models/submission/CISSubmissionBuilder.scala

```diff
@@ -30,19 +30,15 @@ object CISSubmissionBuilder {
 
   val anUpdateCISSubmission: CISSubmission = CISSubmission(
```

Comment:
Please create a builder for PeriodData. It should not be here.

Reply:
updated